apps/apple/FluentUITester/macos/FluentUITester-macOS/ViewController.m

```diff
@@ -0,0 +1,22 @@
+#import "ViewController.h"
+#import "AppDelegate.h"
+
+#import <React/RCTRootView.h>
+
+@implementation ViewController
+
+- (void)viewDidLoad {

```

Comment:
Looks more like a loadView than a viewDidLoad? clients frequently get this wrong so we shouldn't provide examples to encourage them. Most recently, deck view got this wrong to absolutely disastrous results